app/controllers/comments_controller.rb

```diff
@@ -17,11 +17,12 @@ def create
 
```

Comment:
Just one file in your `Files changed` which is the `comments_controller.rb` as you can see in the screenshot.
![Screenshot from 2022-12-03 13-30-38](https://user-images.githubusercontent.com/62849872/205441091-6bb4e51d-19a4-43a0-a9be-291b396658bf.png)